apps/user_status/lib/Migration/Version1030Date20210809144824.php

```diff
@@ -33,7 +33,7 @@
 /**
  * @package OCA\UserStatus\Migration
  */
-class Version2301Date20210809144824 extends SimpleMigrationStep {
+class Version1030Date20210809144824 extends SimpleMigrationStep {
```

Comment:
If the app was `1.3.0` the expected name would be:
```suggestion
class Version1003Date20210809144824 extends SimpleMigrationStep {
```